### ISSUE TITLE
Ubuntu 14.04 Installation via Package Repository Fails apt-get "E: Unable to locate package powershell"

### ISSUE THREAD

**Issue by gunther3613** [PowerShell/PowerShell#6530]
Steps to reproduce
------------------

```
Follow the "Installation via Package Repository - Ubuntu 14.04" instructions (https://github.com/PowerShell/PowerShell/blob/master/docs/installation/linux.md#installation-via-package-repository---ubuntu-1404)
```

Expected behavior
-----------------

``` 

# Install PowerShell
sudo apt-get install -y powershell
...(apt-get finds package package powershell and installs)...

```

Actual behavior
---------------

```
# Install PowerShell
sudo apt-get install -y powershell
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package powershell

```

Environment data
----------------

```

>lsb_release -a
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.5 LTS
Release:        14.04
Codename:       trusty

>uname -a
Linux 4.4.0-116-generic #140~14.04.1-Ubuntu SMP Fri Feb 16 09:25:49 UTC 2018 i686 i686 i686 GNU/Linux

>sudo apt-key list
...
pub   2048R/BE1229CF 2015-10-28
uid                  Microsoft (Release signing) <gpgsecurity@microsoft.com>
...

>sudo apt-get update |grep microsoft
Hit https://packages.microsoft.com trusty InRelease
Hit https://packages.microsoft.com trusty/main amd64 Packages
Get:1 https://packages.microsoft.com trusty/main Translation-en_US
Ign https://packages.microsoft.com trusty/main Translation-en_US
Ign https://packages.microsoft.com trusty/main Translation-en
```

**Comment by TravisEz13** [comment]
I just verified that these instruction work:  https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md#installation-via-package-repository---ubuntu-1404 

Note: many steps are missing from your repro.

**Comment by gunther3613** [comment]
I apologize, I forgot to specifically mention that I running am I running 32bit (x86) Ubuntu 14.04. It is the 'uname -a' (kernel version output) but didn't explicitly call that out. 

> uname -a
>Linux 4.4.0-116-generic #140~14.04.1-Ubuntu SMP Fri Feb 16 09:25:49 UTC 2018 **i686 i686 i686** GNU/Linux

I figured out the issue why it is not working because the package is a 64bit package. A 64bit package will never install on a 32 bit system

I copied this from the manual download steps from the install instructions to point out that this a 64bit package.

> Download the Debian package powershell_6.0.2-1.ubuntu.14.04_**amd64**.deb from the releases page onto the Ubuntu machine.
